relay-profiling/src/lib.rs

```diff
@@ -118,10 +118,15 @@ mod utils;
 
 const MAX_PROFILE_DURATION: Duration = Duration::from_secs(30);
 
+/// Unique identifier for a profile.
+///
+/// Same format as event IDs.
+pub type ProfileId = EventId;
```

Comment:
I added this type alias to make the return type more obvious in function signatures.

Reply:
Shouldn't we just new type it here instead?

Then it's actually a separate type and cant get mixed